docs/2.9.0/docs/getting-started/index.rst

```diff
@@ -3,8 +3,8 @@
 
 .. _new-quickstart:
 
-Getting Started with Daml
-#########################
+Getting Started
+###############
```

Comment:
with Daml should remain here.

Reply:
Do we have standards for when to add "Daml" to the title and when not to? Since this entire document is dedicated to Daml, I find it unclear when we should or shouldn't include "Daml" in the title without making it sound redundant.

Reply:
That seems like a good solution.